hubspot-style/src/main/java/com/hubspot/immutables/utils/WireSafeEnum.java

```diff
@@ -52,7 +53,8 @@
  *    of the field will need to get updated
  */
 @JsonDeserialize(using = Deserializer.class)
-public final class WireSafeEnum<T extends Enum<T>> {
+public final class WireSafeEnum<T extends Enum<T>> implements Serializable {
+  private static final long serialVersionUID = 42000404L;
```

Comment:
this is literally me rolling my hand across the number keys, we should probably pick something better (if that's a thing?)